dll/win32/msgina/shutdown.c

```diff
@@ -91,6 +93,37 @@ GetShutdownReasonUI(VOID)
 //    return (VersionInfo.wProductType == VER_NT_WORKSTATION) ? FALSE : TRUE;
 }
 
+static
+BOOL
+GetShutdownLunaUI(VOID)
+{
+    DWORD dwValue, dwSize;
+    HKEY hKey;
+    LONG lRet;
+
+    lRet = RegOpenKeyExW(HKEY_LOCAL_MACHINE,
+                         L"Software\\Microsoft\\Windows\\CurrentVersion\\Reliability",
```

Comment:
For your information the registry key path is `SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Reliability` with `HKEY_LOCAL_MACHINE` as registry hive and `LogonType` as registry value in Windows XP. While MSGINA is the module that queries such value WINLOGON is the actual component that initiates and its registry data as well as `LogonType`.

Reply:
TY for information! 😋